controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiHandler.java

```diff
@@ -211,17 +211,4 @@ private static Inspector bodyInspector(HttpRequest request) {
         return mapper.apply(object.field(name));
     }
 
-    private static String valueOf(Role role) {
-        switch (role.definition()) {
-            case tenantOwner:           return "tenantOwner";
-            case tenantAdmin:           return "tenantAdmin";
-            case tenantOperator:        return "tenantOperator";
-            case applicationAdmin:      return "applicationAdmin";
-            case applicationOperator:   return "applicationOperator";
-            case applicationDeveloper:  return "applicationDeveloper";
-            case applicationReader:     return "applicationReader";
-            default: throw new IllegalArgumentException("Unexpected role type '" + role.definition() + "'.");
```

Comment:
I would keep both these, as they are used in different serialisation contexts.

Reply:
Added it back.